docs/resolvers.md

```diff
@@ -113,19 +113,17 @@ public class QueryType
 }
 ```
 
-The above example declares a type named `Query` with one field called `greetings` of the type `String` that always returns `foo`. Like with the schema-first approach we can create types that are not explicitly bound to a specific .NET type like in the above example.
+The above example declares a type named `QueryType` with one field called `greetings` of the type `String` that always returns `foo`. Like with the schema-first approach we can create types that are not explicitly bound to a specific .NET type like in the above example.
```

Comment:
The type is named "Query"... maybe we should clarify the sentence to 

`The above example declares a schema type named `Query` since the name refers to the GraphQL type not the .NET type.

Reply:
Great point! I will change to that.